lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -6,6 +6,7 @@ SQLAlchemy==1.0.15
 mercurial==3.7.3
 numpy==1.9.2
 pycrypto==2.6.1
+pyuwsgi==2.1.dev0+gx1.c6ee1f6
```

Comment:
@nsoranzo I appreciate you bringing this and I don't know the answer to the naming question. Given the difficulty of this - do you think resolving that is important prior to merging this? I think we can just create an issue to track this and go with what is here for now - but obviously we will wait if you think we need to.